src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,40 @@
+.nav {
+  &__list {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    list-style: none;
+    font-size: 12px;
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    height: 60px;
+    align-items: center;
+    text-decoration: none;
+    color: $color-main-accent;
+    font-weight: 500;
+
+    &:hover {
+      color: $color-blue-accent;
+    }
+
+    &--active {
+      color: $color-blue-accent;
+
+      &::after {
+        position: absolute;
+        align-self: flex-end;
```

Comment:
it is `position: absolute` you need to position it with bottom, top or left ...

Reply:
Got it.